### ISSUE TITLE
block-no-empty reports incorrect column position

### ISSUE THREAD

**Issue by Arcanemagus** [stylelint/stylelint#1544]
> Describe the issue. Is it a bug or a feature request (new rule, new option, etc.)?

An invalid column is reported when a SugarSS file has an empty block.

> Which rule, if any, is this issue related to?

`block-no-empty`

> What CSS is needed to reproduce this issue?

```sugarss
.a
```

> What stylelint configuration is needed to reproduce this issue?

```json
{
  "extends": "stylelint-config-standard",
  "rules": {
    "declaration-block-trailing-semicolon": null
  }
}
```

> Which version of stylelint are you using?

* `stylelint@6.7.1`
* `stylelint-config-standard@9.0.0`

> How are you running stylelint: CLI, PostCSS plugin, Node API?

* CLI with `stylelint --syntax sugarss foo.sss` on Node v6.2.2
* Node API (`linter-stylelint` Atom plugin on Atom v1.9.0-beta0 ~ Node v5.10.0)

> Does your issue relate to non-standard syntax (e.g. SCSS, nesting, etc.)?

Yes, it's related to SugarSS.

> What did you expect to happen?

The proper column number to be reported.

> What actually happened (e.g. what warnings or errors you are getting)?

Instead of column 2 (?), the error is being reported as being on column 12, possibly a mapping issue between the SugarSS source and the CSS being processed?

![image](https://cloud.githubusercontent.com/assets/427137/16396521/f95c7fe2-3c73-11e6-8965-ae3b3b765052.png)

A directory that reproduces this can be found here: [stylelint_sugarss_block-no-empty.zip](https://github.com/stylelint/stylelint/files/335882/stylelint_sugarss_block-no-empty.zip)

**Comment by jeddy3** [comment]
@Arcanemagus Thanks for the detailed report.

> the error is being reported as being on column 12, possibly a mapping issue between the SugarSS source and the CSS being processed?

Agreed. Such a high number as 12 makes me think this might be an upstream issue with `sugarss`.

Opening this out to anyone who uses the sugarss syntax and this rule, and wants to see this fixed.

**Comment by jeddy3** [comment]
Done in https://github.com/stylelint/stylelint/pull/1547
